setup.cfg

```
@@ -19,4 +19,8 @@ known_first_party = uvicorn,tests
 known_third_party = click,does_not_exist,gunicorn,h11,httptools,pytest,requests,setuptools,urllib3,uvloop,watchgod,websockets,wsproto,yaml
 
 [tool:pytest]
-addopts = --cov=uvicorn --cov=tests -rxXs
+addopts = -rxXs
+
+[coverage:run]
+omit=venv/*
+source=uvicorn,tests
```

From experience, `include` produces coverage reports with nicer path information (if you ever use something like Codecov).
```suggestion
include = uvicorn/*, tests/*
```

---

yes indeed, done